server/server.js

```diff
@@ -92,13 +121,20 @@ app.post('/api/sign-up', async (req, res, next) => {
     const params = [username, hashedPassword, firstName, lastName, email];
     const result = await db.query(sql, params);
     const [user] = result.rows;
+    const sqlOrders = `
+    insert into "orders" ("userId")
+    values ($1)
+    returning *;
+    `;
+    const paramsOrders = [user.userId];
+    await db.query(sqlOrders, paramsOrders);
```

Comment:
You create the order when the user is created. That's not bad. When a user places an order and you take the payment, you might want to create a new order for them so they can order other stuff.